src/sentry/lang/native/symbolicator.py

```diff
@@ -76,10 +77,10 @@ def run_symbolicator(stacktraces, modules, project, arch, signal, request_id_cac
 
                 rv.raise_for_status()
                 json = rv.json()
-                metrics.incr(
-                    'events.symbolicator.response.%s' % json['status'],
-                    tags={'project_id': project_id}
-                )
+                metrics.incr('events.symbolicator.response.%s', tags={
```

Comment:
What are you putting into the %s now?

Reply:
I'm putting my full qualities as a software engineer in there.